apps/ecommerce/frontend/src/components/MultipleResources.tsx

```diff
@@ -40,10 +41,10 @@ const MultipleResources = () => {
 
   const mockValue: ExternalResourceLink = {
     sys: {
-      urn: 'gid://products/8191006998814',
+      urn: crypto.randomUUID(),
```

Comment:
I hesitate to care about this in the context of this PR, but I don't like the pattern of envisioning a UUID or other opaque string here. It's too early to formalize it but let's at least start off in the mock value with something that mimics what we eventually expect to sea, even e.g

```
`http://api.somemagento.com/products/${crypto.randomUUID()}`
```

Reply:
yeah this is 100% just for generalization of the mock data purposes currently. definitely not wanting to keep this around either!